lib/v2/picuki/utils.js

```diff
@@ -3,7 +3,7 @@ const puppeteerGet = async (url, browser) => {
     // await page.setExtraHTTPHeaders({ referer: host });
     await page.setRequestInterception(true);
     page.on('request', (request) => {
-        request.resourceType() === 'document' ? request.continue() : request.abort();
+        request.resourceType() === 'document' || request.resourceType() === 'script' ? request.continue() : request.abort();
```

Comment:
This route also works if `script` is blocked. Are there any reasons to whitelist it?